src/fragments/lib/auth/js/advanced.mdx

```diff
@@ -169,74 +169,63 @@ const App = () => {
 
 ### Google sign-in (React)
 
-```js
+
+```jsx
```

Comment:
Is it worth adding a note/callout letting folks know why this looks like a workaround?

Reply:
I don't think this is really a workaround - this is the way that Identity Pool / social login federation is supposed to work.